src/reducers/index.js

```diff
@@ -1,6 +1,3 @@
-import { combineReducers } from 'redux';
 import { charsReducer } from './starWarsReducer';
 
-export default combineReducers({
-  charsReducer
-});
+export default charsReducer
```

Comment:
it's important to know how this would look if you _did_ have multiple reducers to pass into the combineReducer functionality: `export default combineReducers({
  charsReducer
});` would work fine, it just adds one more object in the object notation chain you need to reference the values on the store. so in your mapStateToProps, instead of getting the characters from `store.characters`, you need to specify the reducer object on the store in the middle, so: `store.charsReducer.characters` and `store.charsReducer.fethcing`, etc. this is because you might have state trees stored in different reducer being passed her so you explicitly show which reducer you're pulling the data from